Cargo.toml

```
@@ -75,8 +75,8 @@ futures-bounded = { version = "0.2.3", path = "misc/futures-bounded" }
 libp2p = { version = "0.53.2", path = "libp2p" }
 libp2p-allow-block-list = { version = "0.3.0", path = "misc/allow-block-list" }
 libp2p-autonat = { version = "0.12.0", path = "protocols/autonat" }
-libp2p-connection-limits = { version = "0.3.0", path = "misc/connection-limits" }
-libp2p-core = { version = "0.41.2", path = "core" }
+libp2p-connection-limits = { version = "0.3.1", path = "misc/connection-limits" }
+libp2p-core = { version = "0.41.1", path = "core" }
```

```suggestion
libp2p-core = { version = "0.41.2", path = "core" }
```

That doesn't seem correct!

---

Thanks. Mustve happened during one of the merges